content/id/docs/concepts/extend-kubernetes/extend-cluster.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Memperluas Kubernetes Cluster Anda
+content_template: templates/concept
+weight: 10
+---
+
+{{% capture overview %}}
+
+Kubernetes sangat mudah dikonfigurasi dan dikembangkan. Sehingga,
+jarang membutuhkan fork atau menambahkan patches ke kode projek Kubernetes.
+
+Panduan ini menjelaskan pilihan untuk menyesuaikan Kubernetes cluster.
+Ini ditujukan kepada {{< glossary_tooltip text="cluster operators" term_id="cluster-operator" >}} yang ingin
+memahami bagaimana menyesuaikan kluster Kubernetes dengan kebutuhan lingkungan kerja mereka.
+Pengembang yang prospektif {{< glossary_tooltip text="Platform Developers" term_id="platform-developer" >}} atau {{< glossary_tooltip text="Contributors" term_id="contributor" >}} Projek Kubernetes juga mendapatkan manfaat dari
+dokumen ini sebagai pengantar apa saja poin dan pola yang ada, untung-rugi dan batasannya.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Gambaran
+
+Pendekatan kostumisasi secara luas dapat dibagi atas *konfigurasi*, yang hanya melibatkan perubahan bendera, konfigurasi file lokal, atau API sumber;
+dan *ekstensi*, yang melibatkan berjalannya program atau layanan tambahan. Dokumen ini utamanya membahas tentang ekstensi.
```

Comment:
```suggestion
dan *ekstensi*, yang melibatkan berjalannya program atau layanan tambahan. Dokumen ini sebagian besar membahas tentang ekstensi.
```

Reply:
Ekstensi is an Indonesian word. 
https://kbbi.kemdikbud.go.id/entri/ekstensi

But to be consistent with translation, should we change it to "perluasan" instead?

Reply:
yeah I agree with the consistency. I'll update it into "perluasan".

Reply:
done